pkg/scalers/rabbitmq_scaler.go

```diff
@@ -269,10 +303,29 @@ func (s *rabbitMQScaler) getQueueInfoViaHTTP() (*queueInfo, error) {
 
 // GetMetricSpecForScaling returns the MetricSpec for the Horizontal Pod Autoscaler
 func (s *rabbitMQScaler) GetMetricSpecForScaling() []v2beta2.MetricSpec {
-	targetMetricValue := resource.NewQuantity(int64(s.metadata.queueLength), resource.DecimalSI)
+	if s.metadata.queueLength > 0 {
```

Comment:
Could you please reformat the code to remove the duplication, the same way it is suggested in `GetMetrics()`